portal/api-tests/static-routes.api-test.js

```diff
@@ -17,16 +17,47 @@ const { get } = require('./create-api').createApi(app);
 
 const allRoles = Object.values(ROLES);
 
-describe('static routes', () => {
-  describe('GET /.well-known/security.txt', () => {
+describe('/.well-known/security.txt', () => {
+  describe('GET', () => {
     withRoleValidationApiTests({
       makeRequestWithHeaders: (headers) => get('/.well-known/security.txt', {}, headers),
       whitelistedRoles: allRoles,
       successCode: 200,
     });
+
+    it('should have all the file properties', async () => {
+      // Act
+      const response = await get('/.well-known/security.txt');
+
+      // Assert
+      expect(response.text).toContain('Contact: https://hackerone.com/7af14fd9-fe4e-4f39-bea1-8f8a364061b8/embedded_submissions/new');
+      expect(response.text).toContain('Contact: https://www.gov.uk/contact/govuk');
+      expect(response.text).toContain('Contact: https://get-a-guarantee-for-export-finance.service.gov.uk/feedback');
+      expect(response.text).toContain('Expires: 2026-01-17T00:00:00.000Z');
+      expect(response.text).toContain('Acknowledgments: https://get-a-guarantee-for-export-finance.service.gov.uk/thanks.txt');
+      expect(response.text).toContain('Preferred-Languages: en');
+      expect(response.text).toContain('Canonical: https://get-a-guarantee-for-export-finance.service.gov.uk/.well-known/security.txt');
+      expect(response.text).toContain('Policy: https://www.gov.uk/guidance/report-a-vulnerability-on-a-ukef-system');
+      expect(response.text).toContain('Hiring: https://www.civilservicejobs.service.gov.uk/');
```

Comment:
Can we split these into individual tests? i.e

```js
it('should contain "Contact" copy', () => {
  
});

it('should contain "Expires" copy', () => {

});

it('should contain "Acknowledgments" copy', () => {

});
```

Would be easier to update or debug if there's a failure